backend/open_webui/apps/webui/routers/auths.py

```diff
@@ -3,6 +3,7 @@
 import time
 import datetime
 import logging
+import httpx
```

Comment:
should be implemented with `aiohttp`

Reply:
I have updated the code to use aiohttp instead of httpx.